src/views/v2/month/calendar-body/day/calendar-events/calendar-event/date.php

```diff
@@ -9,13 +9,15 @@
  *
  * @link {INSERT_ARTCILE_LINK_HERE}
  *
- * @version 4.9.9
+ * @version TBD
  *
  * @var WP_Post $event The event post object with properties added by the `tribe_get_event` function.
  *
  * @see tribe_get_event() For the format of the event object.
  */
-$time_format = tribe_get_time_format();
+
+$time_format      = tribe_get_time_format();
+$display_end_date = $event->dates->start->format( 'H:i' ) !== $event->dates->end->format( 'H:i' );
```

Comment:
I think we should be using `$event->dates->start_display->format( 'H:i' )` here instead? Is there an equivalent for `end_display`? @lucatume ?

Reply:
There is `$event->dates->end_display`.